persistence/scaladsl/src/main/scala/com/lightbend/lagom/internal/scaladsl/persistence/AbstractPersistentEntityRegistry.scala

```diff
@@ -108,6 +109,11 @@ abstract class AbstractPersistentEntityRegistry(system: ActorSystem, injector: I
         throw new IllegalArgumentException(s"[${entityClass.getName} must first be registered")
     }
 
+  override def refFor2[C, P <: PersistentEntity[C, _, _]: ClassTag](entityId: String): PersistentEntityRef[C] = {
```

Comment:
@jroper The attempt with abstract type members breaks down completely (as far as I can see) when trying to use it with the PersistentEntityTestDriver. I think that approach is going down a rabbit hole. You can see the problems by `testkit-scaladsl/test:compile` in https://github.com/patriknw/lagom/tree/wip-scaladsl-ClassTag-NOT-WORKING-patriknw

I was thinking that we could at least try ClassTag for `refFor` and that would require two parameters like this, kind of repeating the command type. That is maybe not bad, but I'm not sure. You can see usage here: https://github.com/lagom/lagom/pull/194/files#diff-5b233e73c9b93c20d145f9ecd318b512R143

WDYT?